packages/nextjs/app/builders/0xcf942c47bc33dB4Fabc1696666058b784F9fa9ef/page.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { NextPage } from "next";
+import { useTheme } from "next-themes";
+import { FaCube, FaEnvelope, FaGithub, FaLinkedin } from "react-icons/fa";
+import { Address } from "~~/components/scaffold-eth";
+
+const name = "Crypto Voyager";
+const githubUrl = "https://github.com/Animesh-Parashar";
+const linkedinUrl = "https://www.linkedin.com/in/animesh-parashar-378659320/";
+const mailTo = "animeshparashar3439@gmail.com";
+const walletAddress = "0xcf942c47bc33dB4Fabc1696666058b784F9fa9ef";
+
+const CryptoVoyager: NextPage = () => {
+  const { resolvedTheme } = useTheme();
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
```

Comment:
Since we're using Next.js, you'll notice the page loads faster without this behavior. Try it yourself and compare the experiences.